### PR TITLE
[ECOS-127]added depth support to uwebsockets

### PR DESCRIPTION
**Pull request by NidaKarimAli**

### CODE REVIEW

File: src/exchange/WebSocket.ts [hunk 1]
```diff
@@ -117,6 +118,46 @@ export class WebSockets {
...
+  ): boolean {
+    if (!this.socketInstance) return false;
+    this.socketInstance.send(
+      JSON.stringify([
+        "SUBSCRIBE",
+        [
+          {
+            e: SOCKET_EVENTS.ORDERBOOK_DEPTH_STREAM_ROOM,
+            p: symbol,
+            d: depth,
```
Comment (yasir7ca) [inline src/exchange/WebSocket.ts:133]: What would happen if depth is empty string, would it subscribe to all depth levels ?

**Review by yasir7ca — APPROVED** [review]

File: src/exchange/WebSocket.ts (hunk 1 above)
Comment (MaazUddin11) [reply src/exchange/WebSocket.ts:133]: I can see in the [dapi PR](https://github.com/fireflyprotocol/dapi-server/pull/1198/files) it defaults to 5. @NidaKarimAli is it not meant to be 20 by default?

Comment (MaazUddin11) [reply src/exchange/WebSocket.ts:133]: I also think we should provide the default value into the client as well as anyone using the client would not know what the default is

Same for the other clients

**Review by MaazUddin11 — APPROVED** [review]

File: src/exchange/WebSocket.ts (hunk 1 above)
Comment (NidaKarimAli) [reply src/exchange/WebSocket.ts:133]: Discussed async on the above comments.

**Pull request merged**